stories/Toolbar.stories.ts

```diff
@@ -43,17 +44,18 @@ class BaseMountedClass extends Vue {
   @Action initializeTreeModel: any
 
   @Mutation configure
-  @Mutation updateIsEditable: any
   @Mutation addEnabledFeature: any
   @Mutation removeEnabledFeature: any
+
+  @builderVuexNamespace.Action setIsEditable: void
 }
 
 // Default
 @Component<any>(
     {
         ...BaseOptions,
       async mounted() {
-        this.updateIsEditable({value: 0})
+        this.setIsEditable(0)
```

Comment:
@poojahpatil90 , let's pass boolean type as defined in IBuilderState

Reply:
Yes, I missed that. I will make the change.